src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/cas2/StatusUpdateService.kt

```diff
@@ -132,6 +135,7 @@ class StatusUpdateService(
               name = newStatus.name,
               description = newStatus.description,
               label = newStatus.label,
+              statusDetails = statusDetails.let { transformer.transformStatusDetailListToDetailItemList(it) },
```

Comment:
No need for this to be a `let`; left over from when statusDetails had a null default value.  Removed and commit squashed.  @Gweaton conversant.